tests/sles4sap/sapconf.pm

```diff
@@ -106,75 +79,23 @@ sub verify_sapconf_service {
     die "Command 'systemctl status $svc' output is not recognized" unless ($output =~ m|$active|s or $output =~ m|$success|s);
 }
 
-sub test_profile {
-    my $profile = shift;
-    assert_script_run "tuned-adm profile_info $profile" if is_sle('15+');
-    assert_script_run "tuned-adm profile $profile";
-    sleep 4;    # add a small sleep to ensure that tuned-adm has correctly switched profile
-    check_profile($profile);
-}
-
 sub run {
     my ($self) = @_;
-    my @tuned_profiles = qw(balanced desktop latency-performance network-latency network-throughput
-      powersave throughput-performance virtual-guest);
-    # Testing 'virtual-host' on a VM isn't very useful and can lead to sporadic timeout issues
-    push @tuned_profiles, 'virtual-host' unless (check_var('BACKEND', 'qemu'));
-    # Add SAP profiles depending of the products and the OS version
-    push @tuned_profiles, is_sle('15+') ? (check_var('SLE_PRODUCT', 'sles4sap') ? qw(sapconf saptune) : 'sapconf') : qw(sap-ase sap-bobj sap-hana sap-netweaver);
 
     $self->select_serial_terminal;
 
     assert_script_run("rpm -q sapconf");
 
-    my $output = script_output "tuned-adm active";
-    $output =~ /Current active profile: ([a-z\-]+)/;
-    my $default_profile = $1;
-    record_info("Current profile", "Current default profile: $default_profile");
-
-    verify_sapconf_service('tuned.service',   'Dynamic System Tuning Daemon');
-    verify_sapconf_service('sapconf.service', 'sapconf') unless ($default_profile eq 'saptune');
+    verify_sapconf_service('sapconf.service', 'sapconf');
     verify_sapconf_service('uuidd.socket',    'UUID daemon activation socket');
     verify_sapconf_service('sysstat.service', 'Write information about system start to sysstat log')
       if is_sle('15+');
 
-    my $statusregex = join('.+', sort(@tuned_profiles));
-    $output = script_output "tuned-adm list";
-    die "Command 'tuned-adm list' output is not recognized" unless ($output =~ m|$statusregex|s);
-
-    $output = script_output "tuned-adm recommend";
-    record_info("Recommended profile", "Recommended profile: $output");
-    die "Command 'tuned-adm recommend' recommended profile is not in 'tuned-adm list'"
-      unless (grep(/$output/, @tuned_profiles));
-
     # Record a softfailure for x86_64 only, as the workarounded bug only happens on this arch
     record_soft_failure 'bsc#1146298 - openQA test fails in sapconf due to timing issue'
```

Comment:
Is this still valid? I've never liked this `if is_x86_64` ... it implies that there's a bug present only based on the architecture instead of actually checking if the bug is present.